mix.exs

```diff
@@ -51,7 +55,7 @@ defmodule Accent.Mixfile do
       {:dataloader, "~> 1.0"},
 
       # GraphQL
-      {:absinthe, "~> 1.4"},
+      {:absinthe, "~> 1.4.13"},
```

Comment:
_semver_ `1.4` includes minor and patch updates. Is there a specific reason to force patch updates only?